common-util/DisplayName/index.jsx

```diff
@@ -1,20 +1,20 @@
 import PropTypes from 'prop-types';
 import Link from 'next/link';
-import { EllipsisMiddle } from '@autonolas/frontend-library';
+import { truncateAddress } from 'common-util/functions';
 
-const DisplayName = ({ actorAddress, account }) => (
+const DisplayName = ({ actorAddress, username }) => (
   <Link href={`/profile/${actorAddress}`}>
-    {actorAddress?.toLowerCase() === account?.toLowerCase() ? 'You' : <EllipsisMiddle>{actorAddress}</EllipsisMiddle>}
+    {username || truncateAddress(actorAddress)}
```

Comment:
@mohandast52 note I reverted to truncateAddress because EllipsisMiddle applies unwanted styling. Would be good to update globally at some point.